### PR TITLE
Clean up zip verification

### PR DESCRIPTION
**Pull request by rkondratowicz**
- throw more appropriate exception when the content of zip file is invalid
- use already existing constant for zip file name in processor

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessor.java [hunk 1]
```diff
@@ -163,13 +161,13 @@ private void processBlob(String blobName, String containerName) {
             ZipEntry entry = null;
 
             while ((entry = zipStream.getNextEntry()) != null) {
-                if (ENVELOPE_ENTRY_NAME.equals(entry.getName())) {
+                if (ZipVerifiers.DOCUMENTS_ZIP.equals(entry.getName())) {
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessor.java]: `ZipVerifiers.ENVELOPES_ZIP` may be ?

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessor.java]: For now I'm just removing the duplication of contants.

The other name is not great either, currently there are those constants in `ZipVerifiers` class:

```java
    public static final String DOCUMENTS_ZIP = "envelope.zip";
    public static final String SIGNATURE_SIG = "signature";
    public static final String INVALID_SIGNATURE_MESSAGE = "Zip signature failed verification";
```

I'd probably move the file names constants to a separate class (like `ZipFileEntries`) and call them `ENVELOPE_ZIP` and `SIGNATURE` respectively.

**Comment by lgonczar** [comment]
Will `InvalidZipArchiveException` cause rejection?

**Comment by rkondratowicz** [comment]
> Will `InvalidZipArchiveException` cause rejection?

good point. Added extra `catch` and new tests

**Pull request merged**